modules/server/src/test/scala/higherkindness/compendium/db/PGHelper.scala

```diff
@@ -40,7 +55,7 @@ abstract class PGHelper extends Specification with ForAllTestContainer {
 
   implicit lazy val CS: ContextShift[IO] = IO.contextShift(global)
 
-  lazy val transactor: Transactor[IO] =
+  implicit lazy val transactor: Transactor[IO] =
```

Comment:
why do you need to make it `implicit`?

Reply:
`PgStorage` apply method needs an implicit transactor and `PgStorageSpec` provides it via `PGHelper`. I changed it to implicit, but another option is to left it untouched as it was and add this to my test:

```
implicit val xa = transactor
private lazy val pgStorage = PgStorage[IO]
```

Reply:
Comment left on `PgStorage.scala`, because I missed that detail.

Reply:
Changed this along with your follow-up suggestion.